Makefile

```diff
@@ -493,8 +496,14 @@ images: #toil-docker
 .PHONY: init-conda
 init-conda:
 	echo "    started init-conda" >> $(LOGFILE)
-	$(CONDA) create -y -n $(VENV_NAME) python=$(VENV_PYTHON) pip=$(VENV_PIP)
-	$(CONDA) activate $(VENV_NAME) && pip install -U -r $(DIR)/etc/venv/requirements.txts
+	# source both bashrc and conda's script to be safe, so the conda command is found
+	source $(DIR)/bin/miniconda3/etc/profile.d/conda.sh && source /home/vagrant/.bashrc \
```

Comment:
should not hardcode sourcing of bashrc path as this may not exist in all cases (ie. non vagrant deployments)